include/NAS2D/Filesystem.h

```diff
@@ -50,16 +57,64 @@ class Filesystem
 
 	void toggleVerbose() const;
```

Comment:
The verbosity flag was removed in a prior commit in master -- I saw no need to keep verbose logging intact.